packages/config/config/devices/0x019b/z-dim2.json

```
@@ -0,0 +1,255 @@
+			"zwaveAllianceId": 4666
+		},
+		{
+			"productType": "0x0002",
+			"productId": "0x2201"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Switch Multilevel",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Power Restore Level",
+			"description": "Range between 0(off) and 99(max)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Return to last level",
+					"value": 100
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "Load Dimming Mode",
+			"description": "Choose the dimming type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Trailing edge",
+					"value": 0
+				},
+				{
+					"label": "Leading edge",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Maximum Dim Level",
+			"description": "Max dim level, range 2-99",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 99,
+			"defaultValue": 90,
+			"unsigned": true
+		},
+		{
+			"#": "12",
+			"label": "Meter Report Threshold",
+			"description": "Threshold for meter report, defined in W",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 250,
+			"defaultValue": 10,
+			"unsigned": true
+		},
+		{
+			"#": "13",
+			"label": "Meter Report Interval",
+			"description": "Time between meter reports, defined in seconds",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 65535,
+			"defaultValue": 780,
+			"unsigned": true
+		},
+		{
+			"#": "2",
+			"label": "Switch On Level",
+			"description": "Dimming level when restored from the OFF state, range 1-99",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Restores last dim level",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Automatic Turn Off",
+			"description": "Timer to automaticly turn off dimmer, defined in seconds",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 86400,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Auto OFF disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Automatic Turn On",
+			"description": "Timer to automaticly turn on dimmer, defined in seconds",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 86400,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Auto ON disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Turn Off Delay Time",
+			"description": "The time it takes before the dimmer turns off after turning it off, defined in seconds",
```

```suggestion
			"label": "Turn Off Delay",
```
and add the unit property